samples/angular/MusicStore/MusicStore.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
```

Comment:
Nit: you can remove the following settings as they are the default value:

PreserveCompilationContext
AssemblyName
OutputType
RuntimeFrameworkVersion